### PR TITLE
ENCD-4523 fix ENSEMBL visualization

### PR DESCRIPTION
**Pull request by yunhailuo**

### CODE REVIEW

**Comment by forresttanaka** [comment]
I only checked the Javascript change.

**Comment by yunhailuo** [comment]
> I only checked the Javascript change.

Thanks!

**Comment by caseylitton** [comment]
Out of date, please get on v82 and rebuild qa demo.

File: src/encoded/static/components/vis_defines.js [hunk 1]
```diff
@@ -115,7 +115,7 @@ export const visOpenBrowser = (dataset, browser, assembly, files, datasetUrl) =>
     }
     case 'Ensembl': {
         if (ASSEMBLY_DETAILS[assembly].ensembl_host) {
-            href = `http://${ASSEMBLY_DETAILS[assembly].ensembl_host}/Trackhub?url=${datasetUrl};species=${ASSEMBLY_DETAILS[assembly].species.replace(/ /g, '_')}`;
```
Comment (caseylitton) [inline src/encoded/static/components/vis_defines.js]: just flagging for rebase.

**Pull request merged**